crate/multiworld-gui/src/main.rs

```diff
@@ -1418,6 +1421,14 @@ impl Application for State {
             Message::ShowLoggingInstructions => if let Err(e) = open("https://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/logging.md") {
                 return cmd(future::err(e.into()))
             },
+            Message::LaunchProject64 => {
+                let emulator_path = self.pj64_script_path.as_ref().expect("emulator path must be set for Project64 version 3");
```

Comment:
A panic (which is what [`Option::expect`](https://doc.rust-lang.org/std/option/enum.Option.html#method.expect) does when `self` is `None`) should only be used for unexpected cases (i.e. application bugs). Since the configuration file is read from disk and can be edited by the user, this needs to be handled more gracefully. In this case, since configuring the `pj64_script_path` entry has not always been part of [the manual install instructions](https://github.com/midoshouse/ootr-multiworld/blob/main/assets/doc/manual-install.md#for-project64), it might be best to handle this silently, i.e. by not displaying the button at all.

Reply:
this makes sense, hence the button will not be displayed if the config file doesnt have the `pj64_script_path `variable